src/components/style/interactive/button.css

```diff
@@ -0,0 +1,83 @@
+.fdsButton {
+  --button-blue: 0, 102, 153;
+  --button-red: 229, 50, 62;
+  border: 0 none;
+  padding: var(--space-half) var(--space-default);
+  border-radius: var(--border-radius-default);
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
I could technically use like 4 utility classes here... but seems a bit excessive.

Reply:
We'll want to revisit the font weight - we currently do not include a 500 weight on cbi-site.

Once that's fixed, we can introduce a `--fontWeight-medium` var.